src/GroupRow.tsx

```diff
@@ -16,7 +16,8 @@ const groupRow = css`
       background-color: var(--rdg-header-background-color);
     }
 
-    > .${cell}:not(:last-child):not(.${cellFrozenLast}) {
+    > .${cell}:not(:last-child):not(.${cellFrozen}),
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/:not#description
>You can negate several selectors at the same time. Example: `:not(.foo, .bar)` is equivalent to `:not(.foo):not(.bar)`.

```suggestion
    > .${cell}:not(:last-child, .${cellFrozen}),
```